Misc/NEWS.d/next/Library/2019-09-13-08-55-43.bpo-38148.Lnww6D.rst

```diff
@@ -0,0 +1 @@
+Add slots to asyncio transports.
```

Comment:
Thanks for the PR @asvetlov.

Could we also slightly mention the purpose/benefit of the slots? Also, it might help to explicit mention that we're specifically talking about the transport *classes*, so it makes more sense as what is being referred to. 

```suggestion
Add slots to :mod:`asyncio` transport classes, which can reduce memory usage.
```

Also added a Sphinx role to link to the module itself. Not 100% needed for a Misc/NEWS entry, but it doesn't hurt.

Reply:
Agree